geoips/schema/sectors_dynamic.yaml

```diff
@@ -0,0 +1,64 @@
+type: object
+required:
+  - interface
+  - metadata
+  - spec
+properties:
+  interface:
+    type: string
+  metadata:
+    type: object
+    required: [name, family, short-description, long-description]
+    properties:
+      name:
+        type: string
+      family:
+        type: string
+      short-description:
+        type: string
+      long-description:
+        type: string
+  spec:
+    type: object
+    properties:
+      area_def_generators:

```

Comment:
Is this an extra field that is not actually included in the YAML files?  Ie, since it is not required, it is ignored, so actually nothing in the spec is actually checked?

That said, at some point in the future (not now!) we probably want to update to create an area_def_generator python module interface (area_def_generator is being hackily used now), and we probably DO want to set the YAML files up this way (similar to how we handle algorithsm, etc in the product_params):
area_def_generator:
  name: clat_clon_resolution_shape
  args:
    projection: ...

Reply:
Well. It is an actual Python module. https://github.com/NRLMMD-GEOIPS/geoips/blob/integration/geoips/interface_modules/area_def_generators/clat_clon_resolution_shape.py

It is not well defined, because I never actually made it into an interface before.  There is not even a type/family defined for it.

There could be other area_def_generators, and there could be other families, but there are not right now.

Reply:
Okay, I'll call this handled for now, but we should revisit creating an interface for this.

Created #163